lighthouse-core/audits/audit.js

```diff
@@ -125,7 +125,12 @@ class Audit {
     if (displayValue === score) {
       displayValue = '';
     }
-
+    let auditDescription = audit.meta.description;
+    if (audit.meta.fallbackDescription) {
+      if (!result.rawValue || (typeof result.rawValue === 'number' && result.rawValue < 100)) {
```

Comment:
Yup, and DOM size is invalid cause of that, one of the things that needed discussion. I change the code accordingly to @patrickhulce 's last comment on the other branch 
![skjermbilde 2017-06-12 kl 21 10 35](https://user-images.githubusercontent.com/16735925/27050670-b7d11d90-4fb3-11e7-83a0-59c0fd379bb9.png)

Evidently I really wanted to use the labels that are used, but that happens later